mix.exs

```diff
@@ -36,14 +36,13 @@ defmodule Avrogen.MixProject do
   defp deps do
     [
       {:accessible, "~> 0.3"},
-      {:credo, "~> 1.7", only: [:dev, :test], runtime: false},
+      {:credo, "== 1.7.1", only: [:dev, :test], runtime: false},
```

Comment:
This was to avoid a big refactor to deal with:

```
┃ [R] ↗ Predicate function names should not start with 'is', and should end in a 
┃       question mark.
┃       lib/avrogen/types.ex:50:7 #(Avrogen.Types.is_primitive?)
```

We can do it one day, but not now